tests/unit/plugins/inventory/test_lxd.py

```diff
@@ -43,8 +43,8 @@ def inventory():
     # Test Values
     inv.data = inv.load_json_data('tests/unit/plugins/inventory/fixtures/lxd_inventory.atd')  # Load Test Data
     inv.groupby = GROUP_Config
-    inv.prefered_container_network_interface = 'eth'
-    inv.prefered_container_network_family = 'inet'
+    inv.prefered_instance_network_interface = 'eth'
+    inv.prefered_instance_network_family = 'inet'
     inv.filter = 'running'
     inv.dump_data = False
```

Comment:
You need something like this here:
```suggestion
    inv.dump_data = False
    inv.typefilter = ...
```